features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/GrokParserStageSequenceBuilder.java

```diff
@@ -198,7 +208,17 @@
 		/**
 		 * Remaining string message.
 		 */
-		message
+		message,
+
+		/**
+		 * Sequence number.
+		 */
+		parmSequenceNum,
```

Comment:
Is there an easy way to make the parameter naming generic so that we don't have to hard-code the parameter names?

Reply:
Could use the text form the grok pattern as the key for the map. That's easy for the map.put(). For the get() you would need to know the literal string for the parameter you want to get since you could no longer reference it here. Probably safe to assume those pattern names will never change and if so then we can drop the grok pattern -> key mapping and get rid of this enum. There's also no getters at the moment to worry about. So yes, I'll change this.